### ISSUE TITLE
New VAOS - Record VCR Cassettes for patient-metadata end-point

### ISSUE THREAD

**Issue by grossb1** [department-of-veterans-affairs/va.gov-team#24591]
As a continuation of the work done in #23879, VCR cassettes need updating to reflect data retrieved from the hosted new VAOS service.

- VCR cassette data reflects actual data hosted by the new VAOS service
- Adjustments are made to the code-base such that vets-api exposes accurate data value/schema to vets-website
- Documentation is updated to reflect any code changes

**Comment by laurenernest** [comment]
blocked by https://issues.mobilehealth.va.gov/browse/VAOSR-2257

**Comment by ajmagdub** [comment]
VAOS-Service release 1.11 postponed
